Header header
uint8 motor
uint8 command
uint8 response
uint8 confirmed
